config/hardware/accelerometers/adxl345_btt_lis2dw.cfg

```diff
@@ -2,7 +2,7 @@
 [mcu adxl]
 serial: /dev/serial/by-id/xxx
 
-[lis2dw]
+[adxl345]
 cs_pin: btt_lis2dw:gpio9
 #spi_bus: spi1a
 spi_software_sclk_pin: btt_lis2dw:gpio10
```

Comment:
`adxl:` in front of the gpio

Reply:
just so i am understanding, 

btt_lis2dw:gpio9

should be

adxl:gpio9

?

Reply:
Any thoughts on axis map? i just took it from their example config.